config/features/glance.yml

```diff
@@ -0,0 +1,15 @@
+features:
+  glance:
+    flags: ""
+    create_kvm_image:
```

Comment:
But yes, it might be usable for other scenarios as well. I think potential solution will become more clear once we start working on such scenarios.